### PR TITLE
[IMP] developer: add section on cookie service

### PR DESCRIPTION
**Pull request by ged-odoo**
and slightly improve notification information

### CODE REVIEW

**Review by aab-odoo — APPROVED** [review]

**Comment by ged-odoo** [comment]
one more for our beloved @odoo/doc-review team...

**Review by Feyensv — APPROVED** [review]
@robodoo delegate+

(I launched a rebuild for the red runbot)

File: content/developer/reference/javascript/services.rst [hunk 1]
```diff
@@ -113,13 +113,53 @@ Reference List
...
+.. code-block:: javascript
+
+   cookieService.setCookie("hello", "odoo");
+
+API
+~~~
+
+.. js:data:: current
+
+   Object representing each cookie and its value if any (or empty string)
```
Comment (Feyensv) [inline content/developer/reference/javascript/services.rst:147]: `WARNING: duplicate data description of current, other data in developer/reference/javascript/services`

duplicate definition of same `js:data` is the cause of the red runbot.

Comment (ged-odoo) [reply content/developer/reference/javascript/services.rst:147]: hmm, that's annoying. Not sure what to do here. I tried the `js:module::` directive (see 
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-js-module), the documentation says that it does not display a heading, which is what I want, but it does not work like that. It displays a ugly heading, just like classes. any idea?

Comment (Feyensv) [reply content/developer/reference/javascript/services.rst:147]: 1) Either encapsulate it as a "false" class
image

2) Begin specifying methods and attributes with respect to their service:

image

```suggestion
.. js:data:: cookieService.current

   Object representing each cookie and its value if any (or empty string)
```

I don't have any other solution keeping the js domain logic.

Comment (ged-odoo) [reply content/developer/reference/javascript/services.rst]: i finallly chose to use the `:noindex:` flag, so it works as expected.

**Comment by ged-odoo** [comment]
@robodoo r+

**Pull request closed without merging**